code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -58,10 +58,12 @@
 
 /obj/item/weapon/reagent_containers/food/snacks/Destroy()
 	QDEL_NULL(dip)
-	var/turf/T = get_turf(src)
+	var/atom/place_to_spawn = loc
+	if(istype(loc,/obj/item/weapon/reagent_containers/food/snacks/customizable/fullycustom)) // consistency
+		place_to_spawn = place_to_spawn.loc
```

Comment:
Please confirm that cutting a custom item on a table won't spawn in the area since loc(loc(src)) may be an area. I wouldn't even keep this extra logic for "consistency" as it breaks consistency with the way other food items are cut.

Reply:
get_turf is probably the safest bet here - update and I'll merge.

Reply:
did it now